dummycloud/src/MqttClient.js

```diff
@@ -19,10 +19,22 @@ class MqttClient {
             this.handleData(data);
         })
     }
-    
+
     initialize() {
-        this.client = mqtt.connect(process.env.MQTT_BROKER_URL);
-        
+        const options = {
+            clientId: "deye_dummycloud" + Math.random().toString(16).slice(2, 10),
```

Comment:
Why?

Reply:
I guess the question is: Why not? If anything breaks on the broker side, this allows identification of the client by name. Just tell me if you want to keep it or if I shall remove it.

Reply:
What I was wondering is if the default clientId is stable

https://github.com/mqttjs/MQTT.js#mqttclientstreambuilder-options

Doesn't seem to be the case

Reply:
What do you mean with stable, exactly?

Reply:
Stable as in stays the same even when restarting the service. e.g. by using some kind of fixed hardware identifier such as a mac address or something like that